settings.ini

```
@@ -15,8 +15,9 @@ language = English
 custom_sidebar = True
 license = apache2
 status = 2
-requirements = numpy, numba, pandas, scikit-learn, quadprog, matplotlib, tqdm, intel-cmplr-lib-rt ; platform_system!="Darwin" and platform_machine=="x86_64"
-dev_requirements = datasetsforecast nbdev statsforecast>=1.0.0 requests scipy pre-commit ruff black pytest pytest-benchmark
+requirements = numpy, numba, pandas, scikit-learn>=1.2, quadprog, matplotlib, narwhals>=1.13.5, intel-cmplr-lib-rt ; platform_system!="Darwin" and platform_machine=="x86_64"
+dev_requirements = datasetsforecast nbdev statsforecast>=1.0.0 requests scipy pre-commit ruff black pytest pytest-benchmark polars
+polars_requirements = polars
```

to ensure numpy compatibility
```suggestion
polars_requirements = polars[numpy]
```